api/app/routers/hfi_calc.py

```diff
@@ -18,12 +19,50 @@
 
 logger = logging.getLogger(__name__)
 
+ageless_cache = "max-age=0"  # don't let the browser cache this
```

Comment:
Ageless cache is just slightly weird for me - could sound like the cache never expires. (the exact opposite)
I'm bad with names, but what about "no_cache" or something like that? As opposed to a vampire cache?